pkgs/molior-client-http/usr/lib/molior/build-script

```diff
@@ -21,6 +21,14 @@ if [ -z "$MOLIOR_SERVER" ]; then
   MOLIOR_SERVER=172.16.0.254
 fi
 
+# Workaround obsolete pxz package on buster
+xzversion=`dpkg -s xz-utils | grep ^Version: | sed 's/^Version: //'`
+if dpkg --compare-versions "$xzversion" lt 5.2.4-1; then
```

Comment:
cat /etc/os-release | grep VERSION_ID= | sed 's/^VERSION_ID=//'
"10"

Reply:
checking the package version is better here...